recipes/cytocad/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cytocad" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 22289cdca9ebf5627dda3acc74f869c4cd73421eed41eb6680505f6e077d6e0a
+
+build:
+  skip: True # [py<36 or osx]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib-base >=2.2.3
+    - numpy >=1.17.3
+    - pybedtools >=0.8.0
+    - python >=3.6
+    - scipy >=1.2.1
+    - bedtools >=2.26.0
+    - samtools >=1.3
+    - pysam >=0.15.3
+    - tagore >=1.1.0
+    - librsvg >=2.50.3
+
+test:
+  imports:
+    - cytocad
+  commands:
+    - cytocad --help
+
+about:
+  home: https://github.com/cytham/cytocad
+  license: "GNU General Public v3 (GPLv3)"
```

Comment:
Can you please use https://spdx.org/licenses/ identifiers.

Reply:
Ok, thanks!